llpc/context/llpcCompiler.cpp

```diff
@@ -777,8 +777,12 @@ Result Compiler::BuildShaderModule(const ShaderModuleBuildInfo *shaderInfo, Shad
 // @param context : Acquired context
 // @param shaderInfo : Shader info of this pipeline
 // @param [out] pipelineElf : Output Elf package
+// @param [out] stageCacheAccesses : Stage cache access result. All elements
```

Comment:
I don't know the technical definitions, but, if you expect that that the parameter is initialized, should it be an in/out parameter?  On the other hand, that initial value is never used, it is just so the caller can differentiate between the values that were set and those that were not.